### PR TITLE
Dev db log viewer

### PR DESCRIPTION
**Pull request by np-at**
Db Log Viewer (currently serilog mssql only).
Added Pagination Property to ApiResponse object to facilitate paginated requests (implemented in /admin/bBlogViewier with MatPaginator)

There may be a slight breaking change if people are relying on serilog db logs going to table Logs2 (renamed to Logs and implemented as a C# efcore object) but I only submitted that pull request a few days ago so ¯\_(ツ)_/¯

It's not perfect: the UI could use a little refinement (specifically table column sizing and automatic refresh interval component) and I'd appreciate it if someone could doublecheck my efcore db log query (I'm pretty sure there's a more efficient way of getting ordered logs but I can't think of it now).

It works though

### CODE REVIEW

File: src/BlazorBoilerplate.CommonUI/Pages/DbLogViewer.razor [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+        await LoadData().ConfigureAwait(true);
+
+    }
+    protected async Task LoadData()
+    {
+        ApiResponseDto apiResponse = null;
+        try
+        {
+            apiResponse = await Http.GetJsonAsync<ApiResponseDto>($"api/DbLog?page={this.pageIndex}&pageSize={this.pageSize}");
```
Comment (enkodellc) [inline src/BlazorBoilerplate.CommonUI/Pages/DbLogViewer.razor]: @np-at  please update this for the latest System.Net.Http.Json:

https://devblogs.microsoft.com/aspnet/blazor-webassembly-3-2-0-preview-3-release-now-available/?


Microsoft.AspNetCore.Blazor.HttpClient | System.Net.Http.Json
-- | --
GetJsonAsync | GetFromJsonAsync
PostJsonAsync | PostAsJsonAsync
PutJsonAsync | PutAsJsonAsync

Comment (GioviQ) [reply src/BlazorBoilerplate.CommonUI/Pages/DbLogViewer.razor]: I have just tested: you can use one version or the other. The Post and Put are a little different, so in my last PR I have preferred Post and Put without "As".
In fact I was interested to solution of https://github.com/dotnet/runtime/issues/31553#event-3170368286
but I still experience this issue with GetFromJsonAsync.
I am still testing.

**Comment by enkodellc** [comment]
@np-at I have only done a code review. Will test out a bit later. I noticed you have added some helper classes / utilities.  These seem to be pretty good additions that most users would want. 

- Refresh Timer 
- Regex Utilities
- Pagination

The main concern I have is the DbLogging migration. Again like most discussions is how BB should handle them. Since it was created I have taken the stance that we have 1 single migration / seed and force a database reset. Basically forcing users who have an existing database to manually refactor.  Maybe the best thing to do is have a single migration for each version..  Honestly EF migrations are new territory for me as well as maintaining a repo like this.

**Comment by np-at** [comment]
Updated to use new httpclient extensions (only the db log view page, I thought it best to leave the others for a different pull/commit in order to make troubleshooting easier down the road if the switchover breaks something)

I left the dbmigration separate mainly because it only applies if you're using mssql as a db backend.  considering that both sqlite and postgres and supported I wanted to make it easy to identify and remove/modify in the event that someone was using a different db. 
I'm hoping to also integrate the serilog postgres sink down the road but I figure this is good for now.

**Comment by enkodellc** [comment]
@np-at just tested and it looks pretty good. I have a couple changes that I will do post-merge. Moving the page under the Admin folder. The last is trivial but to keep it consistent, move the pagination below the table. 

Thanks so much for your contributions! I hope you continue to support Blazor Boilerplate and it has helped you out with your project.

**Pull request merged**